### PR TITLE
tmz 3.1.0 feature symbol deu kurrent

### PR DESCRIPTION
**Pull request by t-m-z**
Kurrent was the predecessor to Sütterlin - Sütterlin is a modern script based on Kurrent that is characterized by simplified letters and vertical strokes.

### CODE REVIEW

File: lib/tools/symbol_tables/_common/assets/kurrent/logo.png
Comment (Mike-3) [inline lib/tools/symbol_tables/_common/assets/kurrent/logo.png:1]: logo
reduced color depth

File: lib/tools/symbol_tables/_common/assets/kurrent/kurrent.zip
Comment (Mike-3) [inline lib/tools/symbol_tables/_common/assets/kurrent/kurrent.zip:1]: [kurrent.zip](https://github.com/S-Man42/GCWizard/files/13854500/kurrent.zip)
reduced color depth

**Review by Mike-3 — CHANGES_REQUESTED** [review]

File: lib/tools/symbol_tables/_common/assets/kurrent/kurrent.zip
Comment (t-m-z) [reply lib/tools/symbol_tables/_common/assets/kurrent/kurrent.zip:1]: yes - but also quality

Comment (t-m-z) [reply lib/tools/symbol_tables/_common/assets/kurrent/kurrent.zip:1]: however, changes accepted

**Pull request merged**